base/include/Command.h

```diff
@@ -166,4 +167,29 @@ class StepCommand : public Command
 	}
 
 
+};
+
+class ValvePassThroughCommand : public Command
```

Comment:
What if the user wants to open the valve forever ? Please improve it to use negative numOfFrames (e.g. -1) to mean that the valve should be open forever (till next command changes). It should be possible to start the valve either open or closed, please add unit tests for both states